SyslogNet.Client/Transport/SyslogLocalSender.cs

```diff
@@ -117,6 +118,16 @@ public void Send(IEnumerable<SyslogMessage> messages)
 			Send(messages, defaultSerializer);
 		}
 
+		public Task SendAsync(SyslogMessage message, ISyslogMessageSerializer serializer)
+		{
+			throw new NotImplementedException();

```

Comment:
Call the sync method and return Task.CompletedTask?

It is fine in this case since it's implementing an interface that has Task returning method but it is more efficient (or only possible) to implement it synchronously. Look at MemoryStream as an example having synchronous implementation of a method advertised as Task returning.